packages/ra-ui-materialui/src/input/AutocompleteArrayInput.spec.tsx

```diff
@@ -888,4 +859,33 @@ describe('<AutocompleteArrayInput />', () => {
             screen.queryByText('Choice is New Kid On The Block')
         ).not.toBeNull();
     });
+
+    it('should show the suggestions when the input value is empty and the input is focused and choices arrived late', () => {
```

Comment:
For some reason, userEvent.clear(input) didn't work, so I move this test to the end to avoid the related error